protocol/blockchain/script.md

```diff
@@ -163,8 +163,8 @@ Op codes marked with **(do not use)** are disallowed and will make a transaction
 | OP_SHA256              | 168  | 0xa8 | in             | hash    | Hashes input with SHA-256. |
 | OP_HASH160             | 169  | 0xa9 | in             | hash    | Hashes input with SHA-256 and then with RIPEMD-160. |
 | OP_HASH256             | 170  | 0xaa | in             | hash    | Hashes input twice with SHA-256. |
-| OP_CODESEPARATOR	      | 171  | 0xab | Nothing        | Nothing | All of the signature checking words will only match signatures to the data after the most recently-executed OP_CODESEPARATOR. | 
-| OP_CHECKSIG	           | 172  | 0xac | sig pubkey	    | true / false     | The entire transaction's outputs, inputs, and script (from the most recently-executed OP_CODESEPARATOR to the end) are hashed. The signature used by OP_CHECKSIG must be a valid signature for this hash and public key. If it is, 1 is returned, 0 otherwise. |
+| OP_CODESEPARATOR	      | 171  | 0xab | Nothing        | Nothing | Makes `OP_CHECK(MULTI)SIG(VERIFY)` set `scriptCode` to everything after the most recently-executed OP_CODESEPARATOR when computing the sighash. | 
```

Comment:
`scriptCode` is kind of implementation specific, would it make sense to reword

> Makes `OP_CHECK(MULTI)SIG(VERIFY)` set `scriptCode` to everything after the most recently-executed OP_CODESEPARATOR

to 

> Makes `OP_CHECK(MULTI)SIG(VERIFY)`use the subset of the script of everything after the most recently-executed OP_CODESEPARATOR

?

Reply:
Yeah I wanted to mention BIP143, but that‘s for SegWit. It‘d also be too much for this opcode. I think your proposal is a good idea.

Eventually we should have an individual Wiki page for every opcode.

Reply:
Updated in https://github.com/BitcoinUnlimited/BitcoinCashSpecification/commit/a358b3ca1d3ebbcbc78478e75031fa1dc60b6278

> Eventually we should have an individual Wiki page for every opcode.

I agree! If in addition to more verbose description, having a small example for each op-code in such a page would be very helpful